pkg/controller/oam/applicationconfiguration/render.go

```diff
@@ -90,10 +95,17 @@ func (r *components) Render(ctx context.Context, ac *v1alpha2.ApplicationConfigu
 		if err != nil {
 			return nil, errors.Wrapf(err, errFmtRenderWorkload, acc.ComponentName)
 		}
-
+		w.SetNamespace(ac.GetNamespace())
 		ref := metav1.NewControllerRef(ac, v1alpha2.ApplicationConfigurationGroupVersionKind)
 		w.SetOwnerReferences([]metav1.OwnerReference{*ref})
-		w.SetNamespace(ac.GetNamespace())
+		// add child resources lists to the workload CR as annotation
+		crMap, err := r.fetchChildResourcesFromWorkloadDefinition(ctx, w)
+		if err != nil {
+			return nil, err
+		}
+		if len(crMap) > 0 {
+			w.SetAnnotations(crMap)
```

Comment:
This will need to align with https://github.com/crossplane/crossplane/pull/1454/files#r423465611